README.md

```diff
@@ -1 +1,34 @@
-# Lucas-Comamala-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Project
+
+GitHub repository for my personal portfolio
+
+## Built With
+
+- HTML
+- CSS
+- Figma template
+
+## Author
+
+👤 **Lucas Comamala**
+
+- GitHub: [@lucascomamala](https://github.com/lucascomamala/)
+- LinkedIn: [Lucas Comamala](https://linkedin.com/in/lucas-comamala/)
+
+## 🤝 Contributing
+
+ - Readme Template from [Microverse Readme Template](https://github.com/microverseinc/readme-template)
+ - Linters for html&css from [Microverse Linters](https://github.com/microverseinc/linters-config)
+ - Portofolio template from [Figma](https://figma.com)
+ 
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- The Microverse Team
```

Comment:
- [ ] [OPTIONAL] Your project is missing a license file. Please add an MIT license file in the root folder of your project. You can use [these template](https://github.com/microverseinc/readme-template/blob/master/MIT.md).